src/modules/users/services/users.service.ts

```diff
@@ -164,6 +173,16 @@ export class UsersService {
 		return this.adapt(updatedUser?.returning[0]);
 	}
 
+	public async updateUserLanguage(id: string, updateLanguage: UpdateUserLangDto): Promise<any> {
+		await this.dataService.execute<
+			UpdateUserLanguageMutation,
+			UpdateUserLanguageMutationVariables
+		>(UpdateUserLanguageDocument, {
+			lang: updateLanguage.language as UpdateUserLanguageMutationVariables['lang'],
```

Comment:
would this also work?
```suggestion
			lang: updateLanguage.language as Locale,
```